src/hooks/useQueue.tsx

```diff
@@ -1,34 +1,44 @@
+import { AxiosResponse } from 'axios'
+import useAxios from 'axios-hooks'
 import { ServerConfigContainer } from 'containers/ConfigContainer'
 import { useMyAxios } from 'hooks/useMyAxios'
 import { Queue } from 'types/backend-types'
 
 const useQueue = () => {
   const { authEnabled } = ServerConfigContainer.useContainer()
-  const { axiosInstance } = useMyAxios()
+  const { axiosManualOptions } = useMyAxios()
+  const [, execute] = useAxios({}, axiosManualOptions)
 
-  const getInstanceQueues = (instanceId: string) => {
-    return axiosInstance.get<Queue[]>(
-      `/api/v1/instances/${instanceId}/queues`,
-      {
-        withCredentials: authEnabled,
-      },
-    )
+  const getInstanceQueues = (
```

Comment:
This is the preferred way of having multiple functions provided by a hook, versus the *breaking the hook up into multiple hooks* pattern seen elsewhere.

Reply:
This comment was before I decided to eliminate the "breaking the hook up into multiple hooks" thing I had done originally. Should have updated this comment. My bad.